website/config.toml

```diff
@@ -164,23 +164,29 @@ url = "/docs/about"
 parent = "about"
 weight = 1
 
+[[languages.en.menus.footer]]
+name = "What is Observability Pipelines?"
+url = "/docs/about/what-is-observability-pipelines"
+parent = "about"
+weight = 2
```

Comment:
I'm curious, do you think it is better to link to a new page on the Vector website here or to link directly to https://www.datadoghq.com/product/observability-pipelines/, like we do in the website header?

From my understanding, `/docs/about/what-is-observability-pipelines` mostly duplicates the information on the linked DD page.

Reply:
@barieom

Reply:
My 5c: I am in favor of having both as long as the Vector page is short and has links to the OP docs.

Reply:
Let's add it here to drive more users from Vector to OP, clearly outlining what differentiates OP so orgs we're converting more orgs like like Instacart, Zendesk, Discover, etc who are/were using Vector looks at and buys OP

Reply:
ah thanks for the clarification! 

Let's link it to `/docs/about/what-is-observability-pipelines`

Reason being for Vector users who end up on vector.dev, we want to be explicit about what the differentiators between OP and Vector are, clearly outlining what OP can do what Vector cannot; case in point, Instacart and Zendesk, who are both heavy Vector users, are planning to buy OP because we released the quotas transform and the ability the enforce a quota across all pipeline instances. We need information somewhere on Vector.dev that calls out the difference between the two. The OP product page on Datadog corpsite, on the other hand, is meant to be a general landing page for everyone, which includes many prospects who've never heard of Vector before 

For example, see the last page of these two marketing product briefs: [general OP product marketing brief, which we share with most prospects](https://drive.google.com/file/d/1evcs0s0749SJeK8QAOygXwaKIUCiSu2U/view) and [OP product marketing brief comparing Vector vs OP, which we only share with prospects using Vector already](https://drive.google.com/file/d/13hRNDjAtvtupE3anAJ61uxZ5jfqkeM0B/view)

Reply:
@dsmith3197 does that work for you?

Reply:
Yeah, that makes sense given the different purposes. Thank you for clarifying @barieom.